site/soc/software/config/versions.yaml

```diff
@@ -41,6 +41,11 @@ data:
         reference: 8f290d3b8d2a8d787f107d05b3be0199d33d7026
         subpath: helm-toolkit
         type: git
+      nova-htk:
+        location: https://git.openstack.org/openstack/openstack-helm-infra
+        reference: 8f290d3b8d2a8d787f107d05b3be0199d33d7026
```

Comment:
please update this commit id here to 5e1ecd9840397bf9e8829ce0d98fcb721db1b74e the same as the value from PR 294. The upstream has uplifted the htk version used in Airship.